### ISSUE TITLE
Depth of view

### ISSUE THREAD

**Issue by crazymedguy** [zsviczian/excalibrain#55]
Would like to request the ability to increase the "Depth" just like in the local graph view.

**Comment by zsviczian** [comment]
Yes, that has been on my mind as well. I hope to get to this sometime.

**Comment by spectralmistral** [comment]
Adding my voice to this one. This would be a game-changer. Seems like there was [another](https://github.com/zsviczian/excalibrain/issues/75#issue-1325176508) request for this as well.

**Comment by ShravanSunder** [comment]
I wanted to say a depth field to see headers would be useful

**Comment by CrackheadoooObsidianUserooo** [comment]
are there any news to that? That would change my workflow immensly into the positive

**Comment by Hyperlooper** [comment]
Just chiming in here to say that this would be extremely useful for me as well. I've evaluated about a dozen tools for graphing, and excalibrain is unparalleled owing to it's ability to sit right next to my unstructured data in obsidian, but being limited to such an extent as far as depth goes does limit the utility somewhat.
